### DISCUSSION TITLE
Release Notes CovPass Version 1.22

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-ios#177] [Announcements]
# Release Notes CovPass Version 1.22

Release 1.22 is currently in final review and is expected to be released within the next couple of hours. 
  
### **Main Features CovPass:**  💉 

•	**Technical Expired Certificates can be scanned into the app [IOS, Android]**:  You can now also scan certificates whose validity date has passed. This is relevant when certificates are to be reissued within the app.

-----------------------------------------------------------------------------------------------------------

### **Bug Fixes CovPass:** 💉 

•	Minor bugfixes where made for both platforms

-----------------------------------------------------------------------------------------------------------
Currently the apps suggest a re-issue for 2/2 vaccinations after a first 1/1 vaccination even though the certificates are not from germany. However we are currently not allowed to re-issue non german DCCs. Besides that the apps currently offer a re-issue even though the recovery certificate is the freshest one within the historic context. However a re-issue for J&J based vaccinations is only appropriate if the recovery is older than the 2/2 vaccination. To not suggest this re-issues anymore we are currently working on a hotfix that will be released asap.
 
-----------------------------------------------------------------------------------------------------------

An outlook for upcoming topics will be published soon!

-----------------------------------------------------------------------------------------------------------

**Thank you all for you support and feedback! Stay safe everyone** 👍 🎊 

Team CovPass and CovPassCheck! 🧑‍⚕️ 👩‍⚕️

**Comment by Ein-Tim** [comment]
When can we expect the hotfix to be released?
